src/component/parser/xml/BpmnXmlParser.ts

```diff
@@ -60,7 +60,7 @@ export default class BpmnXmlParser {
   parse(xml: string): BpmnJsonModel {
     let model: BpmnJsonModel;
     try {
-      model = this.xmlParser.parse(xml);
+      model = this.xmlParser.parse(xml) as BpmnJsonModel;
```

Comment:
**question:** I don't understand using the "as..." part is required.
The model variable has a type and the parse method is supposed to return `BpmnJsonModel`

This pattern repeats in several places in the current proposal and I don't understand the related changes neither.